frontend/src/App.tsx

```diff
@@ -16,31 +15,22 @@ function App() {
   return (
     <>
       <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
+        <a href='https://docs.squid.cloud' target='_blank'>
+          <img src={squidLogo} className='logo' alt='Squid Cloud logo' />
         </a>
       </div>
       <h1>Vite + React</h1>
-      <div className="card">
+      <div className='card'>
         <button onClick={() => handleClick()}>
           count is {data?.count || 0}
         </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
       </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
     </>
-  )
+  );
 
   async function handleClick() {
```

Comment:
I would move this above the return statement, typically the return comes at the end of the function.